packages/core/src/domain/configuration.ts

```diff
@@ -215,18 +226,44 @@ function getEndpoint(type: string, conf: TransportConfiguration, source?: string
   return `https://${host}/v1/input/${conf.clientToken}?${parameters}`
 }
 
+function getHost(type: EndpointType, conf: TransportConfiguration) {
+  if (NEW_INTAKE_DOMAIN_ALLOWED_SITES.indexOf(conf.site) !== -1) {
+    return `${type}.browser-intake-${conf.site}`
+  }
+  const oldTypes = {
+    [EndpointType.LOGS]: 'browser',
+    [EndpointType.RUM]: 'rum',
+    [EndpointType.TRACE]: 'public-trace',
+  }
+  return `${oldTypes[type]}-http-intake.logs.${conf.site}`
+}
+
+function getIntakeUrls(conf: TransportConfiguration) {
+  if (conf.proxyHost) {
+    return [`https://${conf.proxyHost}/v1/input/`]
+  }
+  const urls = [
+    `https://rum-http-intake.logs.${conf.site}/v1/input/`,
+    `https://browser-http-intake.logs.${conf.site}/v1/input/`,
+    `https://public-trace-http-intake.logs.${conf.site}/v1/input/`,
+  ]
+  if (NEW_INTAKE_DOMAIN_ALLOWED_SITES.indexOf(conf.site) !== -1) {
+    urls.push(
+      `https://rum.browser-intake-${conf.site}/v1/input/`,
+      `https://logs.browser-intake-${conf.site}/v1/input/`,
+      `https://trace.browser-intake-${conf.site}/v1/input/`
+    )
+  }
+  return urls
+}
```

Comment:
This looks great, nice and simple. One question (and forgive me if it was already discussed beforehand): do we need to store it in the configuration? Could this list be inlined in the `isIntakeRequest`? From my understanding, storing it in the configuration is a way to cache it, but I'm not sure that we should use the configuration as a cache.

Reply:
I agree that this is not the best way to store the intake URLs. But the other solution would be to store the `userConfiguration.site` inside the configuration object and let `isIntakeRequest` compute the URLs on every call. Do you think we should go that way?

Reply:
A third solution may be to use a strategy similar to `configuration.isEnabled`: instead of storing `intakeUrls` in the configuration object, we would implement a `configuration.isIntakeUrl` method. WDYT @bcaudan ?

Reply:
That make sense to me, this function is already in the configuration file so it was begging to be inside configuration 🙂

Reply:
I implemented `configuration.isIntakeUrl`